### ISSUE TITLE
Missing tooltip for the new drag and drop icon

### ISSUE THREAD

**Issue by bobbingwide** [WordPress/gutenberg#25537]
**Describe the bug**
In Gutenberg 9.0.0 there's a new icon in the toolbar. It's a double vertical ellipsis or number 6 domino/Ddice.
I can't get a Tooltip to appear for it.
Even after applying the fix for #25380
 

**To reproduce**
1. Go the block editor.
2. Create at least two blocks.
3. Ensure Top toolbar is not selected.
4. Select a block so that the toolbar is displayed
5. Hover over the new icon.

**Expected behavior**
A tooltip should be displayed.

**Screenshots**
Note: In this screenshot the mouse pointer was over the icon. The cursor had changed from the one finger pointer (Link select) to the open hand (pan).
![image](https://user-images.githubusercontent.com/2474435/93908070-130d1400-fcf6-11ea-8c43-954fda3a08e1.png)

**Editor version (please complete the following information):**
- WordPress version: [e.g: 5.3.2] 5.5.1
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? [e.g: "gutenberg plugin", "default"] Gutenberg 9.0.0.
- If the Gutenberg plugin is installed, which version is it? [e.g., 7.6]

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows
 - Browser [e.g. chrome, safari] Chrome

**Comment by ntsekouras** [comment]
Hey @bobbingwide - thanks for reporting that!
